sdk/python/kfp/dsl/_python_component.py

```diff
@@ -47,3 +49,50 @@ def _python_component(func):
     return func
 
   return _python_component
+
+def component(func):
```

Comment:
It's confusing that the component annotation is under _python_component module. Could you either change the module name or change it to another module file?

Reply:
Good idea. I will move it to _component.py then because changing the module name will break the python_component codes.

Reply:
Move both _python_component and _component to _component.py